lib/task/propel/propelGenerateSlugsTask.class.php

```diff
@@ -258,14 +258,14 @@ private function getStringToSlugify($row, $table)
    */
   private function getInformationObjectStringToSlugify($row)
   {
-    if (null === $basis = QubitSetting::getByName('slug_basis_informationobject'))
+    if (null === $basis = sfConfig::get('app_slug_basis_informationobject'))
     {
       return $row[1]; // Fall back to title as the slug basis if no setting present
     }
 
     // Note: pull reference codes from ES, as hydrating an ORM object and building the inherited
     // reference code on-the-fly is not performant.
-    switch ($basis->getValue())
+    switch ($basis)
```

Comment:
The same in here, you can remove the previous if and do it directly in the switch with a default value:

```
switch (sfConfig::get('app_slug_basis_informationobject', QubitSlug::SLUG_BASIS_TITLE))
```